### PR TITLE
Update ESC calibration docs

### LINKED ISSUE #222: Update ESC Calibration documentation
### Describe the issue

* [x] Clear to remove propellers before starting

### Affected File

_No response_

### Affected Line

_No response_

### PR DESCRIPTION
**Pull request by jamie-54**
## Main Changes

Improved documentation on ESC calibration procedures 

## Associated Issues

- Closes #222 

## Tests

N/A

### CODE REVIEW

File: hardware/README.md [hunk 1]
```diff
@@ -9,8 +8,9 @@ This sets it to calibration mode.
 After a particular delay the signal to the ESC is set back to low which completes the calibration. 
 
 ### Step to calibrate ESC
+${\color{red}REMOVE \space PROPELLERS \space FROM \space QUADCOPTER}$
```
Comment (alphaville) [inline hardware/README.md]: Try 
```html
<p style="color:red;">REMOVE PROPELLERS FROM QUADCOPTER</p>
```

Comment (jamie-54) [reply hardware/README.md]: That doesn't seem to work for me

Comment (alphaville) [reply hardware/README.md]: Hmm... you're right! Then maybe use `\rm`:

```md
${\color{red}\rm REMOVE \space PROPELLERS \space FROM \space QUADCOPTER!}$
```

which will look like this:

${\color{red}\rm REMOVE \space PROPELLERS \space FROM \space QUADCOPTER!}$

Alternatively:

⚠️⚠️⚠️ **REMOVE PROPELLERS!** ⚠️⚠️⚠️

Comment (jamie-54) [reply hardware/README.md]: How about now?

File: hardware/README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ This sets it to calibration mode.
 After a particular delay the signal to the ESC is set back to low which completes the calibration. 
 
 ### Step to calibrate ESC
-${\color{red}REMOVE \space PROPELLERS \space FROM \space QUADCOPTER}$
+⚠️⚠️⚠️ ${\color{red}\rm REMOVE \space PROPELLERS \space FROM \space QUADCOPTER!}$ ⚠️⚠️⚠️
```
Comment (alphaville) [inline hardware/README.md]: ⚠️ should be `:warning:`

**Review by alphaville — APPROVED** [review]
LGTM

**Pull request merged**